### DISCUSSION TITLE
What is version `2.x` and how it differs from release?

### DISCUSSION THREAD

**Discussion by 532910** [antonym/netboot.xyz#1189] [Q&A] [answered]
While `2.66` is the latest release available on gh:
```
% strings 2.0.66/netboot.xyz.efi | grep 'set version'
set version 2.0.66
```
[Quick Start](https://netboot.xyz/docs/quick-start) offers to [download](https://netboot.xyz/downloads/) a `2.x` one:
```
% strings 2.x/netboot.xyz.efi | grep 'set version' 
set version 2.x
```

Is `2.x` a daily build? Why quick start doesn't offers to download the latest release?

**Comment by antonym** [comment, marked as answer]
2.x is a rolling build. The released versions are a fixed point in time that happen every month or so, but 2.x will promote the latest version of operating systems as they come in.

As releases like Arch happen monthly, rolling immediately promotes them whereas there may be some lag in a released version. It provides a better experience for someone looking for the latest version of an upstream operating system.
